doc/how_to/combine_recordings.rst

```diff
@@ -1,4 +1,4 @@
-Combine Recordings in SpikeInterface
+Combine recordings in SpikeInterface
```

Comment:
We may want to add a note in the docs explaining the capitalization rules for tutorial titles.  I think first word or "important" words are both acceptable (in English at least) so if we are picking a style it should be mentioned in our style guide somewhere.

Reply:
I'll add a note. Currently we had a random style :)

Reply:
done in last commit :)